Improve contrast between header link and primary background

yenaing-oo opened the pull request:
Closes https://github.com/NationalCentreTruthReconciliation/Secure-Record-Transfer/issues/314

![image](https://github.com/user-attachments/assets/bd9edd90-2fc3-4e58-a1ea-85ad35552722)

I changed the header link colour to white.

danloveg reviewed (changes requested):
This fixed this issue, but created another one. The breadcrumb links are now too light and don't contrast against the light grey background:

![Screenshot 2025-01-21 084227](https://github.com/user-attachments/assets/bf31aa9d-7c74-41ea-9369-d9df301f1759)

I also noticed that the other colour I pointed to in that image has low contrast against the red background.

danloveg commented:
Just double checked and that header colour issue went away, I'm seeing a bold white text in light mode now which is what's expected.

yenaing-oo commented:
> Just double checked and that header colour issue went away, I'm seeing a bold white text in light mode now which is what's expected.

Thanks for checking - glad it works.

I made some more changes to fix the styling in "auto" mode. I figured out the cause of the hybrid styling in "auto" mode - it was because that particular css variable (`--selected-row`) was not set for the dark theme settings, which meant it used whatever was provided in the light theme in `auto`.

In auto:
![image](https://github.com/user-attachments/assets/7614c2be-cee3-47df-9b39-981d2445b6c1)

In light:
![image](https://github.com/user-attachments/assets/ac0a5179-e1fc-4a41-bc95-3ef9a73ec3fe)


For `dark` mode however, even though `--selected-row` wasn't included in the dark mode settings, it used the default yellow colour, which resulted in a greenish colour that you might be used to be by now:
![image](https://github.com/user-attachments/assets/0948c091-4fe9-493b-8137-512dd5aa3c0f)

I fixed it by explicitly setting that variable for the dark mode settings [here](https://github.com/NationalCentreTruthReconciliation/Secure-Record-Transfer/pull/323/commits/fe971f542d7dbfc31722d5065268f53c02ae003d). I also removed the media query since the dark mode styles can be set in using `html[data-theme="dark"], :root {}`

danloveg commented on bagitobjecttransfer/recordtransfer/templates/admin/base_site.html:
```diff
@@ -29,18 +30,31 @@
+            }
+
+            #user-tools a, #logout-form button {
+                border-bottom: 1px solid rgb(194, 194, 194);
```
All the other colours are in hex, this one looks out of place here

```suggestion
                border-bottom: 1px solid #c2c2c2;
```

danloveg commented on bagitobjecttransfer/recordtransfer/templates/admin/base_site.html:
```diff
@@ -29,18 +30,31 @@
             --default-button-bg: #3c83a3;
             --button-bg: #325261;
             --breadcrumbs-bg: #292929;
+            --selected-row:rgb(78, 99, 109);
```
```suggestion
            --selected-row: #4e636d;
```
